src/backend/distributed/commands/multi_copy.c

```diff
@@ -268,79 +277,40 @@ CopyFromWorkerNode(CopyStmt *copyStatement, char *completionTag)
 	NodeAddress *masterNodeAddress = MasterNodeAddress(copyStatement);
 	char *nodeName = masterNodeAddress->nodeName;
 	int32 nodePort = masterNodeAddress->nodePort;
-	char *nodeUser = CurrentUserName();
-
-	if (XactModificationLevel > XACT_MODIFICATION_NONE)
-	{
-		ereport(ERROR, (errcode(ERRCODE_ACTIVE_SQL_TRANSACTION),
```

Comment:
Let's be cautious with removing these until we know that it is correct to do so. (e.g. DDL followed by COPY would still cause trouble).